crates/test-utils/src/authority.rs

```diff
@@ -119,60 +114,3 @@ pub fn test_authority_aggregator(
 pub fn get_client(config: &ValidatorInfo) -> NetworkAuthorityClient {
     NetworkAuthorityClient::connect_lazy(config.network_address()).unwrap()
 }
-
-/// Submit a certificate containing only owned-objects to all authorities.
-pub async fn submit_single_owner_transaction(
-    transaction: Transaction,
-    configs: &[ValidatorInfo],
-) -> Vec<TransactionInfoResponse> {
-    let certificate = make_certificates(vec![transaction]).pop().unwrap();
-
-    let mut responses = Vec::new();
-    for config in configs {
-        let client = get_client(config);
-        let reply = client
-            .handle_certificate(certificate.clone())
-            .await
-            .unwrap();
-        responses.push(reply);
-    }
-    responses
-}
-
-/// Keep submitting the certificates of a shared-object transaction until it is sequenced by
-/// at least one consensus node. We use the loop since some consensus protocols (like Tusk)
-/// may drop transactions. The certificate is submitted to every Sui authority.
-pub async fn submit_shared_object_transaction(
```

Comment:
Are we sure we don't need this anymore?

Reply:
Yes there is a duplicate in transaction.rs